stats.py

```diff
@@ -494,6 +538,14 @@ def get_revision(
     }
 
 
+# If the passed value is a boolean, then we convert it to the
+# string "1" or "0". Otherwise we return it as a string.
+def convert_value_to_string(value):
+    if isinstance(value, bool):
+        return str(int(value))  # "1" for True, "0" for False
+    return str(value)  # fallback: convert everything else to string
+
```

Comment:
Python styling nit: When leaving a comment about a function, you should use a Python docstring instead of leaving a comment inline above the function. This will look better in most IDEs using Treesitter for syntax highlighting, as well as integrate with the LSP signature help function. You leave a short one-line description of the function, and then add more details below. Also, leaving comments above the line instead of inline is preferred. For this method, it would look like this:

```suggestion

def convert_value_to_string(value):
    """Coerce transaction values to string.
    
    If the passed value is a boolean, then we convert it to the string 
    "1" or "0". Otherwise we return it as a string.
    """
    if isinstance(value, bool):
        # "1" for True, "0" for False
        return str(int(value))
    
    # fallback: convert everything else to string
    return str(value)  
```